executor/seqtest/seq_executor_test.go

```diff
@@ -793,6 +793,8 @@ func (s *seqTestSuite) TestCartesianProduct(c *C) {
 	plannercore.AllowCartesianProduct.Store(true)
 }
 
+// the entry count is removed, so the batch insert/delete may not used anymore.
```

Comment:
If this test is not used any more, please remove it directly.

Reply:
This PR is still WIP, plz wait.